aten/src/ATen/native/mkldnn/Linear.cpp

```diff
@@ -578,6 +591,8 @@ mkldnn_scaled_mm(const Tensor& mat1, const Tensor& mat2,
     args.insert({DNNL_ARG_ATTR_SCALES | DNNL_ARG_SRC, src_scales_t});
   }
   args.insert({DNNL_ARG_ATTR_SCALES | DNNL_ARG_WEIGHTS, wei_scales_t});
+  ideep::tensor dst_scales_t = ideep::tensor(ideep::scale_t(1, output_scale));
+  args.insert({DNNL_ARG_ATTR_SCALES | DNNL_ARG_DST, dst_scales_t});
```

Comment:
Do we need to check `output_scale != 1.0f` here?

Reply:
Ok, I will follow the output_scale in qconv and add the check here.

Reply:
Done.